Gemfile

```diff
@@ -48,6 +42,12 @@ group :development, :test do
   %w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
     gem lib, git: "https://github.com/rspec/#{lib}.git", branch: 'main'
   end
+  gem 'rubocop-rails', require: false
```

Comment:
This should be also placed into `development` group

Reply:
fixed on the other PR @wenderjean